src/minecraft/kotlin/xyz/wagyourtail/unimined/internal/minecraft/task/RemapSourcesJarTaskImpl.kt

```diff
@@ -0,0 +1,68 @@
+package xyz.wagyourtail.unimined.internal.minecraft.task
+
+import xyz.wagyourtail.unimined.api.mapping.MappingNamespaceTree
+import xyz.wagyourtail.unimined.api.minecraft.MinecraftConfig
+import xyz.wagyourtail.unimined.util.deleteRecursively
+import java.nio.file.Path
+import java.util.jar.JarEntry
+import java.util.jar.JarOutputStream
+import java.util.zip.Deflater
+import javax.inject.Inject
+import kotlin.io.path.*
+
+abstract class RemapSourcesJarTaskImpl @Inject constructor(provider: MinecraftConfig): RemapJarTaskImpl(provider) {
```

Comment:
I don't like this extending RemapJar, imo it'd make more sense if you made an `AbstractRemapJarTask` that they both implemented. mainly so things like `tasks.withType` don't accidentally target both when they don't mean to

Reply:
makes sense